### DISCUSSION TITLE
Welcome to MountWizzard4 Discussions!

### DISCUSSION THREAD

**Discussion by mworion** [mworion/MountWizzard4#96] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ACMBakx** [comment]
Hi Michel,
Just read your message at the 10M forum.
How do I update to the latest version?
Pressing install 1.0.0 did not work.
Message: "Could not install update installation"
I now have 0.220.9b2

Arie

**Comment by mworion** [comment]
Hi Arie,
If everything went fine the last session we had, you should be able to use the internal updater again. I think I did a repair in your setup. If that does not work, please let me know. 
Michel

Von meinem iPad gesendet

**Comment by ACMBakx** [comment]
Pressing install 1.0.0 did not work.
Message: "Could not install update installation"

**Comment by ACMBakx** [comment]
Running "update.bat" did work.
Do I need to replace my current run.bat and update.bat?

**Comment by mworion** [comment]
Hi Arie,
you we’re right using the update.bat. If you have a running environment, you could keep the scripts. In case of any problems I would recommend using the new ones as they write more log info in case of errors. 

Michel

Von meinem iPad gesendet

**Comment by kzkcpj** [comment]
Hello,
I use PegasusAstro on a PC with Windows 10, if I understand correctly, I can only configure Peasus using INDI, do you plan to add an ASCOM driver?
PS: Great software !!!!

Thank you

Jakub

**Comment by mworion** [comment]
Jakub,

the Pegasus ASCOM support is on the list, might take some time. I have to readd the AutoDew support, Pegasus broke some firmware versions ago.

Michel

**Reply by kzkcpj** [reply]
great news :) thank you, I still have a question, when I release the virtual keyboard, it writes Conecting .... and it doesn't connect, what am I doing wrong? (Mount is connected correctly)

**Comment by mworion** [comment]
Did you set the mount IP ?
Did you enable the webservice in the mount, you could try: http://ip of your mount. If that works, MW4 should also connect.
Michel

**Reply by kzkcpj** [reply]
Ok I'll try, thank you

**Reply by kzkcpj** [reply]
it already works, it must be turned on using the keypad web server (setting / Net Services / Web Interface: On)
thank you for the advice

**Comment by georgesblum** [comment]
Hi Michel
I started to use your MW app with my GM1000HPS, to begin I use the mount, the simulatorV2 for the camera, MGbox,  the PC, 
i have every thing green solver is ASTAP, I tried Astromety.net but I miss the lib. and I am not enough inside to understand the whole picture...Later...When I run a model the plate solving never complete but the process continue, is it normal with the simulator of the camera ? Is ASTAP ok as a solver ?  Thanks a lot and thanks.   George

**Reply by mworion** [reply]
Hi George,
I personally use ASTAP and I have no issues with it. On which system do you work? If you are using the simulator (on ASCOM) it does not generate star images, which could be solved. Instead there is a tool called ascom sky simulator. If you are on INDI, the CCD Simulator needs the coordinates the mount is pointing to. Otherwise you get stars in the image, but wrong location.
Michel

**Comment by georgesblum** [comment]
Dear Michael,

Unfortunately with Ascom the soft (last version) is crashing when i try to built model…It stops after the first point and it is pending…and i always have issues with plate solving Astap.
I probably do something wrong. To bad because it is exactly what i need. I do not understand what i am doing wrong…
I downloaded H18 H17 V17 W08 and Hyperleda Index for windows and installed them to my computer.

Best regards,

georges


> Le 9 juin 2022 à 09:34, Michael ***@***.***> a écrit :
> 
> 
> Hi George,
> I personally use ASTAP and I have no issues with it. On which system do you work? If you are using the simulator (on ASCOM) it does not generate star images, which could be solved. Instead there is a tool called ascom sky simulator. If you are on INDI, the CCD Simulator needs the coordinates the mount is pointing to. Otherwise you get stars in the image, but wrong location.
> Michel
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/mworion/MountWizzard4/discussions/96#discussioncomment-2911371>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZF4RHRHSW7M3NKWVCKFVN3VOGM7FANCNFSM4U4IFFRQ>.
> You are receiving this because you commented.

OVH SMB Image-innée
***@***.***
http://www.gbmb.fr

**Comment by jgusb** [comment]
Hi Michel,

Having a permanent observatory setup I only remodel every 6 months or so.  I am suddenly having trouble getting Plate Solving to work within MW4. I use Pinpoint and that solves okay within SGPro, so I know that the reference files are all there.  However, when I try to set this up within MW4 the File Path and Index remain red.
![MW4 picture](https://github.com/user-attachments/assets/fba77fd1-6f13-4b40-886e-4d4473d1e28e)

I am also finding that MW4 will not update the Earth Rotation data - it says that no updater is available, which there is.

Perhaps I have forgotten something since I last did these tasks.  Can you help?

Thank you

John

**Comment by ACMBakx** [comment]
Hi John,I am sorry, but I cannot help you with Pinpoint. I am a ASTAP user.Currently I have my own issues connecting my camera to MW4. Hope you get it fixed quickly. ArieVerstuurd vanaf mijn iPadOp 31 jul 2024 om 17:17 heeft jgusb ***@***.***> het volgende geschreven:﻿
Hi Michel,
Having a permanent observatory setup I only remodel every 6 months or so.  I am suddenly having trouble getting Plate Solving to work within MW4. I use Pinpoint and that solves okay within SGPro, so I know that the reference files are all there.  However, when I try to set this up within MW4 the File Path and Index remain red.
MW4.picture.png (view on web)
I am also finding that MW4 will not update the Earth Rotation data - it says that no updater is available, which there is.
Perhaps I have forgotten something since I last did these tasks.  Can you help?
Thank you
John

**Comment by mworion** [comment]
Hi John,
MW4 does not support Pinpoint (despite the fact that I was pinpoint user as well in the past). I would recommend ASTAP as Arie told.
Michel
